powerfulseal/policy/action_pods.py

```diff
@@ -16,9 +16,26 @@
 import random
 import re
 
+from powerfulseal import makeLogger
 from powerfulseal.metriccollectors.collector import POD_SOURCE
 from .action_nodes_pods import ActionNodesPods
 
+class StartHostAction():
+    """ A little helper class to start hosts in cleanup """
+    def __init__(self, driver, host, logger=None):
+        self.driver = driver
+        self.host = host
+        self.logger = logger
```

Comment:
if logger is `None` dont we want to make a logger?

Reply:
Good catch, thanks!